samples/tfm_integration/psa_level_1/CMakeLists.txt

```diff
@@ -13,13 +13,22 @@ elseif (CONFIG_TFM_BL2_FALSE)
     set(TFM_BL2_ARGUMENT BL2 False)
 endif()
 
+list(APPEND EXTRA_ARGUMENTS EXTRA_ARGS)
+#only build tfm_s binary
+list(APPEND EXTRA_ARGUMENTS -DBUILD_S_ONLY=True)
+if(${TFM_TARGET_PLATFORM} STREQUAL "LPC55S69")
+    #create a smaller release binary
+    list(APPEND EXTRA_ARGUMENTS -DCMAKE_BUILD_TYPE=Minsizerel)
+endif()
```

Comment:
Should there be a Kconfig setting for this ?
```
config TFM_BUILD_TYPE_<something>
```

I don't have the overview of number of `CMAKE_BUILD_TYPE`s in TFM to decide if it would be most appropriate with a free-form string, or specific choices like: `TFM_BUILD_TYPE_MIN_SIZE`.

Then `lpcxpresso55s69_ns_defconfig` could specify its setting.